cmd/sonobuoy/app/args.go

```diff
@@ -87,6 +90,12 @@ func AddKubeconfigFlag(cfg *Kubeconfig, flags *pflag.FlagSet) {
 	flags.Var(cfg, "kubeconfig", "Path to explicit kubeconfig file.")
 }
 
+// AddPluginFlag describes which plugin's images to interact
+func AddPluginFlag(cfg *string, flags *pflag.FlagSet) {
+	// The default is the empty string (look in the environment)
```

Comment:
Not sure I follow this comment; I dont see any code looking in the environment for a default value or anything.  Seems like the default should just be `e2e` since it is the only value for now and our main use case?